core/src/components/input/input.ios.scss

```diff
@@ -8,6 +8,15 @@
   font-size: $input-ios-font-size;
 }
 
+/**
+ * The no-fill inputs have a bottom border,
+ * so we only need to round the top left and
+ * top right edges.
+ */
+:host .input-wrapper {
```

Comment:
I opted to duplicate that styles since the bottom border styles are only set when using ios/md. As a result, the entire component will be rounded when using base components